### PR TITLE
Common codebase to reproduce results

### PR DESCRIPTION
**Pull request by zeynepakkalyoncu**
- [ ] Made minor edits to the robust04 branch to support training
- [ ] Split functions into multiple files for readability
- [ ] Added option to load local BERT model and tokenizer for servers without Internet access or memory restrictions
- [ ] Updated instructions for running

### CODE REVIEW

File: util.py [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-def load_pretrained_model_tokenizer(model_type="BertForSequenceClassification", device="cuda", chinese=False):
+def load_pretrained_model_tokenizer(model_type="BertForSequenceClassification",
+                                    local_model=None, local_tokenizer=None,
+                                    device="cuda", chinese=False):
     # Load pre-trained model (weights)
-    if chinese:
-        base_model = "bert-base-chinese"
-    else:
-        base_model = "bert-base-uncased"
+    if local_model is None:
```
Comment (Victor0118) [inline util.py]: what happened when local_model is not None?

Comment (zeynepakkalyoncu) [reply util.py]: Good catch - `local_model` was supposed to be called `base_model`. I fixed it in the latest commit.

**Review by Victor0118 — APPROVED** [review]
LGTM

**Comment by Victor0118** [comment]
LGTM

**Pull request merged**